playground/server.js

```diff
@@ -397,7 +397,7 @@ function parseCommandLine() {
 }
 
 function start(config) {
-  setupApp(config).listen(config.httpPort, function() {
+  setupApp(config).listen(process.env.PORT || config.httpPort, function() {
```

Comment:
This is for live playground right ? It might be good to specify on top of this file, that it is meant to be run on our local computer, but also on heroku for the live playground ?

Reply:
Sorry, I'm not sure to understand your comment :thinking: